go/service/main.go

```diff
@@ -547,6 +548,28 @@ func (d *Service) runHomePoller(ctx context.Context) {
 	return
 }
 
+func (d *Service) runMerkleAudit(ctx context.Context) {
+	if !libkb.IsMobilePlatform() {
+		d.G().Log.Debug("MerkleAudit disabled (not desktop, not starting)")
+		return
+	}
+
+	eng := engine.NewMerkleAudit(d.G(), &engine.MerkleAuditArgs{})
+	go func() {
```

Comment:
Why in the background?

Reply:
Copied this from PerKeyUserUpgradeBackground, looks like this is a leftover from before a BackgroundTask code reorganization.